src/AP.AzureADAuth/Services/AuthenticationService.cs

```diff
@@ -37,8 +37,7 @@ public async Task<AuthenticationResult> LoginAsync()
             if (result is null)
             {
                 var accounts = await _client.GetAccountsAsync();
-                var builder = _client.AcquireTokenInteractive(_configuration.Scopes)
-                                     .WithAccount(accounts.FirstOrDefault());
+                var builder = _client.AcquireTokenInteractive(_configuration.Scopes);
```

Comment:
did MSAL remove the requirement for an account?

Reply:
I went through the [MSAL Android consideration](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-net-xamarin-android-considerations). No account requirement is mentioned over there while acquiring token.